scss/atoms/buttons/_mixins.scss

```diff
@@ -36,6 +36,10 @@
 		@include btn-hover($btn-colors);
 	}
 
+	&:hover {
+		text-decoration: none;
+	}
```

Comment:
.hover already inherits the text-decoration: none from the btn mixin up above (line 8).  This is dealing with browser defaults overwriting the pseudo class functionality.